openeo/extra/job_management/__init__.py

```diff
@@ -540,6 +542,16 @@ def _job_update_loop(
                         stats["job_db persist"] += 1
                         total_added += 1
 
+        # Act on jobs
+        for job, metadata in jobs_error:
```

Comment:
`metadata` might be a bit confusing. I think we use something like `row` elsewhere